src/main/kotlin/com/sainsburys/k2zpl/command/barcode/BarCode39.kt

```diff
@@ -34,32 +38,29 @@ internal data class BarCode(
  * @param data data encoded in the barcode
  * @param x horizontal position
  * @param y vertical position
- * @param barcodeType Barcode type
  * @param orientation The orientation of the barcode.
- * @param checkDigit Whether to include a check digit.
+ * @param checkDigit Mod-43 check digit
```

Comment:
The previous `@param` description "Whether to include a check digit." was pretty clear to me. I admit I have no context on the ZPL programming language, but have we considered merging the old and new descriptions together like so?

```kotlin
/**
 * @param checkDigit whether to include a Mod-43 check digit
 */
```